admin/lib/workarea/api/admin/engine.rb

```diff
@@ -13,6 +13,7 @@ class Engine < ::Rails::Engine
         end
 
         config.after_initialize do
+          Workarea::Api::Admin::DateIndexes.load
```

Comment:
This will need to be done in a `to_prepare` to respect development environment reloading.

Reply:
wouldn't that defeat the whole point of it? The LocalizedActiveFields bits in base aren't in a `to_prepare`

Reply:
Yeah that's a mistake then, it would suffer the same problem.